frontend/src/routes.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Route, Router } from "react-router-dom";
+import { createBrowserHistory } from "history";
+import App from "./screens/Home";
+import Navbar from "./components/Navbar";
+import NewProductPage from "./screens/NewProduct";
+import ShoppingCart from "./screens/ShoppingCart";
+import Login from "./screens/Login";
+import ForgotPassword from "./screens/ForgotPassword";
+import Register from "./screens/Register";
+import ManageProduct from "./screens/ManageProduct";
+
+const history = createBrowserHistory();
+
+export default (
+  <Router history={history}>
+    <Navbar />
+    <br />
```

Comment:
https://github.com/oursky/presentation-microservices-kotlin/pull/23#discussion_r327597451

Reply:
we can use margin-bottom instead of a bunch of <br>s